.github/workflows/ci.yaml

```diff
@@ -112,8 +112,21 @@ jobs:
           - elixir: 1.15.x
             otp: 25.x
             os: ubuntu-20.04
-          # Something changed in otp 26 for seed and single test started producing failures
-          # it does not affect functionality, only CI
-          # - elixir: 1.15.x
-          #   otp: 26.x
-          #   os: ubuntu-20.04
+          - elixir: 1.16.x
+            otp: 24.x
+            os: ubuntu-20.04
```

Comment:
I wonder if some of the old Elixir versions should be removed from CI?

In a few other projects, they've kept Elixir 1.15, 1.16, and 1.17 on OTP 25, 26, and 27 respectively.

Also, wondering if it's really necessary to test on multiple versions of Ubuntu or if that could be outside of the matrix?

(but that's just my opinion -- not the maintainer here)

Reply:
@nathany-copia I follow https://hexdocs.pm/elixir/compatibility-and-deprecations.html